onnxruntime/test/providers/cpu/math/clip_test.cc

```diff
@@ -57,8 +57,8 @@ TEST(MathOpTest, Clip) {
                          -5.0f, 0.0f, 5.0f,
                          -5.0f, 2.0f, 5.0f});
 
-  // nGraph does not support Clip opset 11 yet.
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kNGraphExecutionProvider});
+  // nGraph and Tensorrt does not support Clip opset 11 yet.
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kNGraphExecutionProvider, kTensorrtExecutionProvider});
```

Comment:
Let's detect this in TRT EP's GetCapability() ?

Reply:
TensorRT parser hasn't been able to detect opset11 ops yet. Will ask nvidia to add.